### PR TITLE
docs: Adding OTF I&EH RFP

### LINKED ISSUE #55: Add OTF RFP
What shall we do with RFPs like this one? https://www.opentech.fund/news/request-for-proposals-impact-engagement-lab/ Should they go in their own  time-bounded section?

### PR DESCRIPTION
**Pull request by RichardLitt**
And made a new temporary section. Closes #55. 

How did you make the ToC @ralphtheninja? It seems to be out of sync.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,6 +216,14 @@ Funded by [MOSS](https://www.mozilla.org/en-US/moss/).
 
 Work of [one of contributors](https://github.com/westnordost/StreetComplete/commits?author=matkoniecz) was sponsored by [NLNet Foundation](https://github.com/ralphtheninja/open-funding#nlnet-foundation), work by the [main author](https://github.com/streetcomplete/StreetComplete/commits?author=westnordost) was funded by [OpenStreetMap foundation](https://blog.openstreetmap.org/2020/07/01/osmf-microgrants-program-congratulations-to-selected-projects/) and [Prototype Fund](https://github.com/streetcomplete/StreetComplete#sponsors).
 
+## Time-bounded Opportunities
+
+### 2024 September 13 - Open Tech Fund Impact and Engagement Lab RFP
+
+OTF is soliciting proposals from service providers, organizations, and individuals interested in providing services to our Impact & Engagement Lab.
```
Comment (matkoniecz) [inline README.md:223]: probably it should be in quotes, or "our" replaced with "their" - this lab is not run by ralphtheninja/open-funding people, AFAIK

(and this one is getting into regular contracts a bit, but mostly remains on topic, I think)

**Review by matkoniecz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RichardLitt) [reply README.md]: Good catch. I put it in a blockquote - I had just copied this from the site.

**Comment by matkoniecz** [comment]
this one seems closer to contracting than open grant call but is quite open-ended rather than extremely specific so seems fine

**Comment by ralphtheninja** [comment]
@RichardLitt I was using `hallmark` which we've used quite a bit in the `level` project. PR inc.

**Pull request merged**